inlang/source-code/sdk/src/loadProject.ts

```diff
@@ -94,23 +90,25 @@ export async function loadProject(args: any): Promise<InlangProject> {
 		)
 	}
 
-	// -- migrate if outdated ------------------------------------------------
+	let fs: Repository["nodeishFs"]
+	if (args.nodeishFs) {
+		// TODO: deprecate
+		fs = args.nodeishFs
+	} else if (args.repo) {
+		fs = args.repo.nodeishFs
+	} else {
+		throw new LoadProjectInvalidArgument(`Repo missing from arguments.`, { argument: "repo" })
+	}
+
 	const nodeishFs = createNodeishFsWithAbsolutePaths({
```

Comment:
maybe add the comment // we narrow the fs down to a NodeishFilesystemSubset because we only need the subset from here